e2e/playwright/tests/command/todo.spec.ts

```diff
@@ -0,0 +1,175 @@
+// ***************************************************************
+import {test, expect} from '@e2e-support/test_fixture';
+
+import TodoMessage, {GithubRHSCategory} from '../../support/components/todo_message';
+import {messages} from '../../support/constants';
+import {getGithubBotDMPageURL, waitForNewMessages} from '../../support/utils';
+import {getBotTagFromPost, getPostAuthor} from '../../support/components/post';
+
+const repoRegex = /https:\/\/github.com\/[\w-]+\/[\w-]+/;
+const prRegex = /https:\/\/github.com\/[\w-]+\/[\w-]+\/pull\/\d+/;
+const issueRegex = /https:\/\/github.com\/[\w-]+\/[\w-]+\/issues\/\d+/;
+
+export default {
+    connected: () => {
+        test.describe('/github todo command', () => {
+            test('from connected account', async ({pages, page, pw}) => {
+                const {adminClient, adminUser} = await pw.getAdminClient();
+                if (adminUser === null) {
+                    throw new Error('can not get adminUser');
+                }
+
+                const dmURL = await getGithubBotDMPageURL(adminClient, '', adminUser.id);
+                await page.goto(dmURL, {waitUntil: 'load'});
+
+                const c = new pages.ChannelsPage(page);
+
+                // # Run todo command
+                await c.postMessage('/github todo');
+                await c.sendMessage();
+
+                // # Wait for new messages to ensure the last post is the one we want
+                await waitForNewMessages(page);
+
+                // # Get last post
+                const post = await c.getLastPost();
+                const postId = await post.getId();
+
+                const todo = new TodoMessage(post.container);
+
+                // * Assert that titles are there for each section
+                // Text are fixed and checked inside todo component handler
+                await expect(todo.getTitle(GithubRHSCategory.OPEN_PR)).toBeVisible();
+                await expect(todo.getTitle(GithubRHSCategory.ASSIGNMENTS)).toBeVisible();
+                await expect(todo.getTitle(GithubRHSCategory.REVIEW_PR)).toBeVisible();
+                await expect(todo.getTitle(GithubRHSCategory.UNREAD)).toBeVisible();
+
+                // * Assert that description are there for each section
+                // Singular/plurals are not taken into account: ticket separated at https://mattermost.atlassian.net/browse/MM-52416
+                // Counters may vary and should be explicitely changed once the test accounts are set
+                await expect(todo.getDesc(GithubRHSCategory.OPEN_PR)).toHaveText('You have 1 open pull requests:');
+                await expect(todo.getDesc(GithubRHSCategory.ASSIGNMENTS)).toHaveText('You have 19 assignments:');
+                await expect(todo.getDesc(GithubRHSCategory.REVIEW_PR)).toHaveText('You have 1 pull requests awaiting your review:');
+                await expect(todo.getDesc(GithubRHSCategory.UNREAD)).toHaveText('You have 1 unread messages:');
```

Comment:
Should we have these numbers defined in a separate file, so its easy to edit in future (if these value changes due to any other situations) and ensure assertions passes, instead of requiring the test to be updated in multiple places? What do you think?

Reply:
💯 to that, I'll add that change in my tests for sidebar/coutners since I need to use them too.